### DISCUSSION TITLE
Use `h` render a component, how to use reactive modelvalue?

### DISCUSSION THREAD

**Discussion by zyyv** [element-plus/element-plus#9361] [Q&A]
# Environment
I have a background to use `h` render  component, not in sfc template. 
And i use a can-control component, i have no way to bind a reative variable to a component. 

# Reproduction

[Element Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uLCBoIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24sIEVsTWVzc2FnZUJveCwgRWxTd2l0Y2ggfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBzaG93TWVzc2FnZSA9ICgpID0+IHtcbiAgY29uc3Qgc3RhdHVzID0gcmVmKGZhbHNlKVxuICBFbE1lc3NhZ2VCb3goe1xuICAgIHRpdGxlOiAnSG93IHRvIGNoYW5nZSBzd2l0Y2ggc3RhdHVzPycsXG4gICAgbWVzc2FnZTogaCgncCcsIG51bGwsIFtcbiAgICAgIGgoJ3NwYW4nLCBudWxsLCBgVHJ5IHRvIGNoYW5nZSBpdDogYCksXG4gICAgICBoKEVsU3dpdGNoIGFzIGFueSwge1xuICAgICAgICAnbW9kZWwtdmFsdWUnOiBzdGF0dXMudmFsdWUsIC8vIEl0J3MgZ3JlYXQgdGhhdCBtb2RlbC12YWx1ZSBjYW4gYWNjZXB0IFJlZiB0eXBlIFxuICAgICAgICAnb25DaGFuZ2UnOiAodmFsKSA9PiB7XG4gICAgICAgIFx0Y29uc29sZS5sb2codmFsKVxuICAgICAgICAgIHN0YXR1cy52YWx1ZSA9IHZhbFxuICAgICAgICB9LFxuICAgICAgfSksXG4gICAgXSksXG4gICAgc2hvd0NhbmNlbEJ1dHRvbjogdHJ1ZSxcbiAgICBjb25maXJtQnV0dG9uVGV4dDogJ09rJyxcbiAgICBjYW5jZWxCdXR0b25UZXh0OiAnQ2FuY2VsJyxcbiAgfSlcbn1cbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxlbC1idXR0b24gdHlwZT1cInByaW1hcnlcIiBAY2xpY2s9XCJzaG93TWVzc2FnZVwiPk9wZW4gTWVzc2FnZUJveDwvZWwtYnV0dG9uPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcbiAgfVxufSIsIl9vIjp7fX0=)

# Steps to reproduce
Clink `Button` in page to open messagebox, and try to change `Switch` status.

# What is Expected?
I hope the reative variable can be change.

# What is actually happening?
Unresponsive ！

**Comment by holazz** [comment]
I created a [PR](https://github.com/element-plus/element-plus/pull/9405) to try to resolve this issue. [Demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgaCB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IEVsTWVzc2FnZUJveCwgRWxTd2l0Y2ggfSBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmNvbnN0IHNob3dNZXNzYWdlID0gKCkgPT4ge1xuICBjb25zdCBzdGF0dXMgPSByZWYoZmFsc2UpXG4gIEVsTWVzc2FnZUJveCh7XG4gICAgdGl0bGU6ICdIb3cgdG8gY2hhbmdlIHN3aXRjaCBzdGF0dXM/JyxcbiAgICBtZXNzYWdlOiAoKSA9PiBoKCdwJywgbnVsbCwgW1xuICAgICAgaCgnc3BhbicsIG51bGwsIGBUcnkgdG8gY2hhbmdlIGl0OiBgKSxcbiAgICAgIGgoRWxTd2l0Y2gsIHtcbiAgICAgICAgbW9kZWxWYWx1ZTogc3RhdHVzLnZhbHVlLFxuICAgICAgICAnb25VcGRhdGU6bW9kZWxWYWx1ZSc6ICh2YWwpID0+IHtcbiAgICAgICAgICBzdGF0dXMudmFsdWUgPSB2YWxcbiAgICAgICAgfSxcbiAgICAgICAgJ29uQ2hhbmdlJzogKHZhbCkgPT4ge1xuICAgICAgICBcdGNvbnNvbGUubG9nKHZhbClcbiAgICAgICAgICBzdGF0dXMudmFsdWUgPSB2YWxcbiAgICAgICAgfSxcbiAgICAgIH0pLFxuICAgIF0pLFxuICAgIHNob3dDYW5jZWxCdXR0b246IHRydWUsXG4gICAgY29uZmlybUJ1dHRvblRleHQ6ICdPaycsXG4gICAgY2FuY2VsQnV0dG9uVGV4dDogJ0NhbmNlbCcsXG4gIH0pXG59XG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtYnV0dG9uIHR5cGU9XCJwcmltYXJ5XCIgQGNsaWNrPVwic2hvd01lc3NhZ2VcIj5PcGVuIE1lc3NhZ2VCb3g8L2VsLWJ1dHRvbj5cbjwvdGVtcGxhdGU+XG4iLCJQbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctOTQwNS1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmZ1bGwubWluLm1qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzL1wiOiBcInVuc3VwcG9ydGVkXCJcbiAgfVxufSIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcInZ1ZVwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9ydW50aW1lLWRvbUBsYXRlc3QvZGlzdC9ydW50aW1lLWRvbS5lc20tYnJvd3Nlci5qc1wiLFxuICAgIFwiQHZ1ZS9zaGFyZWRcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0B2dWUvc2hhcmVkQGxhdGVzdC9kaXN0L3NoYXJlZC5lc20tYnVuZGxlci5qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTk0MDUtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiLFxuICAgIFwiQGVsZW1lbnQtcGx1cy9pY29ucy12dWVcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsImVsZW1lbnQtcGx1cy5qcyI6ImltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcbmltcG9ydCBFbGVtZW50UGx1cyBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmxldCBpbnN0YWxsZWQgPSBmYWxzZVxuYXdhaXQgbG9hZFN0eWxlKClcblxuZXhwb3J0IGZ1bmN0aW9uIHNldHVwRWxlbWVudFBsdXMoKSB7XG4gIGlmIChpbnN0YWxsZWQpIHJldHVyblxuICBjb25zdCBpbnN0YW5jZSA9IGdldEN1cnJlbnRJbnN0YW5jZSgpXG4gIGluc3RhbmNlLmFwcENvbnRleHQuYXBwLnVzZShFbGVtZW50UGx1cylcbiAgaW5zdGFsbGVkID0gdHJ1ZVxufVxuXG5leHBvcnQgZnVuY3Rpb24gbG9hZFN0eWxlKCkge1xuICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgIGNvbnN0IGxpbmsgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaW5rJylcbiAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgIGxpbmsuaHJlZiA9ICdodHRwczovL3ByZXZpZXctOTQwNS1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcydcbiAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignZXJyb3InLCByZWplY3QpXG4gICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgfSlcbn0iLCJfbyI6eyJzaG93SGlkZGVuIjp0cnVlLCJzdHlsZVNvdXJjZSI6Imh0dHBzOi8vcHJldmlldy05NDA1LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

**Reply by zyyv** [reply]
Thanks !
